platforms/vercel-with-redwood/.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
maybe also remove this? not really important to this repo since it's just one of many